### PR TITLE
Added Storybook setup procedure for local environment

### LINKED ISSUE #6: Update README with Storybook installation and usage instructions
#### Reason/Context
@yashsehgal are you able to edit the README file to include the Storybook setup instructions? It doesn't have to be anything crazy, just so people here know what to do to review changes. Thanks!

**Note:** I will be updating the README to include more information about contributing to the design system, but this will work for now.

### PR DESCRIPTION
**Pull request by yashsehgal**
**Description**

- I have added basic steps to follow, in order to start storybook locally.
- I have also created an automated gitpod-config(`.gitpod.yml`) file for someone who uses gitpod for development.

**Related issue(s)**
Resolves #6

### CODE REVIEW

**Review by mcturco — CHANGES_REQUESTED** [review]
Thank you! Just a couple of suggestions :)

File: README.md [hunk 1]
```diff
@@ -1,70 +1,13 @@
...
+# AsyncAPI's Design System
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Environment Setup
 
-## Available Scripts
+### How to setup storybook locally
 
-In the project directory, you can run:
+- You can also start the storybook environment locally to test components. Make sure you've installed all the dependencies from `package.json`.
```
[the remark this answers is not in the document]
Comment (yashsehgal) [reply README.md:7]: Good point, will add the steps for it.

**Comment by yashsehgal** [comment]
I have done the following changes @mcturco 
You can take a look.

**Review by mcturco — APPROVED** [review]
Great everything looks good to me! Thank you for making those edits.

**Comment by fmvilas** [comment]
Love that we're going with Storybook from the beginning 👏

**Pull request merged**